tests/conftest.py

```diff
@@ -126,7 +126,7 @@ def temporary_windows_drive(repo_dir):
     if set_up_result == 0:
         raise RuntimeError("Failed to mount windows drive!")
 
-    yield new_drive
+    yield os.path.join(new_drive, os.sep)
```

Comment:
Why is os.sep needed?

Reply:
good question, especially that this change is not a part of this change, I think I might did something wrong when rebasing

Reply:
I must have did something wrong when rebasing

Reply:
Sorry, I was wrong, this is because new drive has a form of `A:` and if you try to join it with some relative path, you end up with `A:some\\path` which is not viable path

Reply:
Let's add a note about it there, since it is not at all obvious.